### PR TITLE
feat(brigade): add kashti chart as dep; install by default

### PR DESCRIPTION
**Pull request by vdice**
Proposed implementation of https://github.com/Azure/brigade-charts/issues/7

Includes Kashti by default in the Brigade chart via the sub-chart approach, preserving the standalone Kashti chart for use cases that might require it.

If 👍 , a follow-up PR will be required to bump the Kashti dependency in `requirements.yaml`.

### CODE REVIEW

**Review by rmb938 — APPROVED** [review]

**Comment by vdice** [comment]
(bumped brigade chart version after rebase with master)

File: charts/kashti/templates/js-configmap.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+{{ $apiServer := include "brigade.apiServer" . }}
```
Comment (technosophos) [inline charts/kashti/templates/js-configmap.yaml:1]: That looks good. Do you still need a port? I can't remember what the current port number is.

**Review by technosophos — APPROVED** [review]
LGTM. I left a comment about the port number, but I'm too rusty to remember whether we switched it to port 80 at some point.

File: charts/kashti/templates/js-configmap.yaml (hunk 1 above)
Comment (vdice) [reply charts/kashti/templates/js-configmap.yaml:1]: Yes, a port is still needed; the default is currently `7745`, hence: https://github.com/Azure/brigade-charts/blob/9e31d7b07c9a1699e61942354dd6d779589ef602/charts/kashti/templates/_helpers.tpl#L19 

Therefore, slightly brittle if this port number ever changes; but hopefully the chances of an update in the brigade chart slipping through without a corresponding update for this chart, now that they are both in the same repo, is slim.

**Pull request merged**